target/linux/lantiq/patches-4.19/4027-NET-MIPS-lantiq-support-fixed-link.patch

```diff
@@ -0,0 +1,84 @@
+--- a/drivers/net/ethernet/lantiq_xrx200.c
++++ b/drivers/net/ethernet/lantiq_xrx200.c
```

Comment:
These patches should contain a proper header with name, authorship, and explanation of the what and why. Look e.g. at https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/patches-4.19/0018-MIPS-pci-ar71xx-convert-to-OF.patch

For your case, it looks to me like you could move all this information from the commit message of your commit (where it is not really helping) to the appropriate patches here. Unfortunately, this hasn't been done too nicely in lantiq so far either, so let's make it better now.